### ISSUE TITLE
Merge into Cats?

### ISSUE THREAD

**Issue by kailuowang** [tpolecat/cats-infographic#13]
The issue in the past was that this relies on proprietary non-free software to edit, but I think the quality provided overweight the issue. ( And maybe we can ask for sponsored license /copy of it. )
So how about merging this into the cats repo?

**Comment by tpolecat** [comment]
Yeah I can do a PR into the cats repo after I get the RC1 changes in.
